colrev/ops/built_in/search_sources/unknown_source.py

```diff
@@ -325,11 +325,69 @@ def __load_md(self, *, load_operation: colrev.ops.load.Load) -> dict:
         return records
 
     def __load_enl(self, *, load_operation: colrev.ops.load.Load) -> dict:
+        enl_mapping = {
+            ENTRYTYPES.ARTICLE: {
+                "T": Fields.TITLE,
+                "A": Fields.AUTHOR,
+                "D": Fields.YEAR,
+                "B": Fields.JOURNAL,
+                "V": Fields.VOLUME,
+                "N": Fields.NUMBER,
+                "P": Fields.PAGES,
+                "X": Fields.ABSTRACT,
+                "U": Fields.URL,
+                "8": "date",
+                "0": "type",
+            },
+            ENTRYTYPES.INPROCEEDINGS: {
+                "T": Fields.TITLE,
+                "A": Fields.AUTHOR,
+                "D": Fields.YEAR,
+                "B": Fields.JOURNAL,
+                "V": Fields.VOLUME,
+                "N": Fields.NUMBER,
+                "P": Fields.PAGES,
+                "X": Fields.ABSTRACT,
+                "U": Fields.URL,
+                "8": "date",
+                "0": "type",
+            },
+        }
+
+        entrytype_map = {
+            "Journal Article": ENTRYTYPES.ARTICLE,
+            "HICSS": ENTRYTYPES.INPROCEEDINGS,
```

Comment:
the unknown_source should not contain fields that are specific to ais_elibrary (including HICS, ...).

Reply:
Okay, updated.